packages/react-instantsearch/src/components/SearchBox.js

```diff
@@ -25,6 +26,9 @@ class SearchBox extends Component {
     onReset: PropTypes.func,
     onChange: PropTypes.func,
 
+    isSearchStalled: PropTypes.bool,
```

Comment:
We could set this value required since `isSearchStalled` will be always pass to the component.

Reply:
From a usage perspective, I would argue that this value is only useful if `showLoadingIndicator` is true and the only value that we really about is `true`. Considering that, do you think it should still be marked as required?

Reply:
We can leave it as it it then, but when we define an optional props it's recommended to always give them a default value. So we can put a default value to `false` on the `isSearchStalled`. WDYT?

Reply:
Ok WFM 👍